src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable react/forbid-prop-types, global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Image, StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { Button } from 'react-native-ui-components';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { VaccineChart } from '../widgets';
+import globalStyles, {
+  SHADOW_BORDER,
+  APP_FONT_FAMILY,
+  SUSSOL_ORANGE,
+  GREY,
+  HAZARD_RED,
+} from '../globalStyles';
+
+const CHEVRON_ICON_STYLE = { size: 18, color: SUSSOL_ORANGE };
+const BREACH_ICON_STYLE = { size: 25, color: HAZARD_RED };
+
+const MILLISECONDS_IN_DAY = 24 * 60 * 60 * 10000;
+const MAX_LOOKBACK_MILLISECONDS = 30 * MILLISECONDS_IN_DAY;
+
+// TODO navigation for menu buttons
+// TODO Localise all strings
+
+export class VaccineModulePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { fridges: [], selectedFridgeId: null };
+  }
+
+  componentWillMount() {
+    const { database } = this.props;
+    const fridges = database.objects('Location').filter(({ isFridge }) => isFridge);
+    const selectedFridgeId = fridges.length > 0 ? fridges[0].id : null;
+    this.setState({ fridges, selectedFridgeId });
+  }
+
+  /* Helper to render menuButton */
+  renderMenuButton = buttonProps => (
+    <Button
+      style={[globalStyles.menuButton, localStyles.menuButton]}
+      textStyle={globalStyles.menuButtonText}
+      {...buttonProps}
+    />
+  );
+
+  renderIcon = (iconName, iconStyle = CHEVRON_ICON_STYLE) => (
+    <Icon style={{ margin: 5 }} name={iconName} {...iconStyle} />
+  );
+
+  /* Render fridge name and chevron-down icon if icon not selected fridge */
+  renderFridgeName = (fridge, isFridgeSelected) => {
+    const onPress = () => this.setState({ selectedFridgeId: fridge.id });
+
+    const { fridgeInfoSectionStyle, fridgeNameTextStyle } = localStyles;
+
+    return (
+      <TouchableOpacity style={fridgeInfoSectionStyle} onPress={onPress}>
```

Comment:
Not a requested change: When a fridge is expanded I think it will keep the `onPress` which does nothing

Reply:
true, but shouldn't break anything, but ideally yeah, shouldn't have onPress on selected fridge